qa/src/test/java/org/eclipse/kapua/service/StepData.java

```diff
@@ -0,0 +1,45 @@
+/*******************************************************************************
+ * Copyright (c) 2011, 2017 Eurotech and/or its affiliates and others
```

Comment:
I know most people just copy it from other files. :wink: 

There is the Eclipse project handbook, which might be helpful in such situations [1] and specifically a section about copyright headers [2]

[1] https://www.eclipse.org/projects/handbook/
[2] https://www.eclipse.org/projects/handbook/#ip-copyright-headers